### PR TITLE
LPS-127933 Fix cursor blinking in step card and EditAppSideBar

### PR DESCRIPTION
**Pull request by gabrielpramos**

### CODE REVIEW

**Comment by gabrielpramos** [comment]
holding this [PR](https://github.com/liferay-app-builder/liferay-portal/pull/227) to be merged into master

File: modules/dxp/apps/app-builder-workflow/app-builder-workflow-web/src/main/resources/META-INF/resources/css/EditApp.scss [hunk 1]
```diff
@@ -269,6 +270,13 @@ $navbarHeight: 120px;
 	}
 }
 
+.dropdown-button-asset,
+.step-card-icon {
+	&.help-cursor {
+		position: absolute;
+	}
+}
+
```
Comment (albinojunior) [inline modules/dxp/apps/app-builder-workflow/app-builder-workflow-web/src/main/resources/META-INF/resources/css/EditApp.scss:279]: @gabrielpramos you can use the not operator `:not(.class)` to apply the `position: absolute` style only for the specific class.

For instance:

```scss

.dropdown-button-asset,
.icon-with-popover,
.step-card-icon {
	&.help-cursor {
	  //common style
	  
	  &:not(.icon-with-popover) {
	          position: absolute;
	  }
	}
```

You could try use this.

Comment (gabrielpramos) [reply modules/dxp/apps/app-builder-workflow/app-builder-workflow-web/src/main/resources/META-INF/resources/css/EditApp.scss:279]: Sure, thanks for reminding. I was thinking I was forgetting something 😊😊

**Comment by rafaprax** [comment]
Hey guys @albinojunior and @gabrielpramos is this change ok now? Can it be forwarded?

Thx

**Comment by gabrielpramos** [comment]
yep, I'm already rebasing and re-testing it before forwarding. :D

**Comment by gabrielpramos** [comment]
ci:forward

**Comment by rafaprax** [comment]
ci:reevaluate:1187401_5368

**Comment by rafaprax** [comment]
ci:forward

**Pull request closed without merging**